ui/app/js/controllers/permissions_controller.js

```diff
@@ -1,161 +1,211 @@
 angular.module("app").controller('PermissionsController',
-function($scope, $routeParams, $location, $timeout, Permissions, Search, $modal, Page, PermissionsRequiredSignoffs) {
```

Comment:
Please revert all of the bulk whitespace changes that you've made...it makes reviewing this very difficult.